srcpkgs/wl-mirror/INSTALL.msg

```diff
@@ -0,0 +1,2 @@
+Optionally install 'pipectl', 'slurp', and either 'rofi' or 'menu' to
+use the helper wl-present script.
```

Comment:
does this script have upstream documentation of these dependencies? if so, i don't see a need for this. if not, this should be a README.voidlinux not INSTALL.msg

Reply:
I would say so: https://github.com/Ferdi265/wl-mirror#script-dependencies.

Reply:
I have just removed the file.